src/components/TaxPicker.tsx

```diff
@@ -54,9 +54,11 @@ type TaxPickerProps = TaxPickerOnyxProps & {
 
     /** The type of IOU */
     iouType?: ValueOf<typeof CONST.IOU.TYPE>;
+
+    onDismiss: () => void;
```

Comment:
I think `goBack` or `navigateBack` name will be more correct and meaningful 😄

Reply:
@dominictb Yeah, but in this case, we execute the `goBack` action not `dismiss` action. As I understand about our codebase, `dismiss` action only be triggered when we want to turn off the modal or keyboard.

Reply:
My concern about the `dismiss` word 😄. It doesn't match with the implementation of this function, this function executes `goBack` action